test/new-e2e/tests/containers/ecs_test.go

```diff
@@ -379,13 +379,14 @@ func (suite *ecsSuite) TestCPU() {
 				`^cluster_name:` + regexp.QuoteMeta(suite.ecsClusterName) + `$`,
 				`^container_id:`,
 				`^container_name:ecs-.*-stress-ng-ec2-`,
-				`^docker_image:ghcr.io/colinianking/stress-ng$`,
+				`^docker_image:ghcr.io/colinianking/stress-ng`,
```

Comment:
Do we now have a random unpredictable suffix at the end of the `docker_image` tag value ?

Reply:
We should be able to predict it. I just wondered if we wanted to still check this. If we plan to regularly bump this image, perhaps it would be easier to not check for the exact value but only for the presence of the tag.